src/main/java/com/google/gcp_variant_transforms/library/VariantToBqUtilsImpl.java

```diff
@@ -43,6 +44,16 @@ public String getReferenceBases(VariantContext variantContext) {
     return nameList;
   }
 
+  public int getStart(VariantContext variantContext, boolean useOneBasedCoordinate) {
+    int oneBasedStartPosition = variantContext.getStart();
+    return useOneBasedCoordinate ? oneBasedStartPosition : oneBasedStartPosition - 1;
+  }
+
+  public int getEnd(VariantContext variantContext, boolean useOneBasedCoordinate) {
+    int oneBasedEndPosition = variantContext.getEnd();
+    return useOneBasedCoordinate ? oneBasedEndPosition : oneBasedEndPosition - 1;
+  }
```

Comment:
0-based or 1-based coordinate only affects the `start_position`. `end_position` value stays the same for both cases but conceptually will be different (inclusive or exclusive).
For more info please refer to [VT-python](https://github.com/googlegenomics/gcp-variant-transforms/commit/2a59483243b26e30eb2b1d4d04036d8473e278a2) or [this post](https://www.biostars.org/p/84686/).